lce-coroutines/src/main/java/com/laimiux/lce/flow/Init.kt

```diff
@@ -0,0 +1,54 @@
+package com.laimiux.lce.flow
+
+import com.laimiux.lce.CE
+import com.laimiux.lce.CT
+import com.laimiux.lce.LC
+import com.laimiux.lce.UC
+import com.laimiux.lce.UCE
+import com.laimiux.lce.UCT
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.onStart
+
+inline fun <C : Any, E> Flow<C>.toUCE(
```

Comment:
Does `Flow` support nullable events?

Reply:
Good catch, yep, will update this to allow for null